### PR TITLE
v3.0 Interface support

### PR DESCRIPTION
**Pull request by mdurrant-b3**
Only handles ethernet interfaces on v30

### CODE REVIEW

**Comment by Cedev** [comment]
Doesn't support enabling the interface.

File: acos_client/v30/interface.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        return self._get(self.url_prefix)
+
+    def get(self, ifnum=None):
+        return self._get(self._url_from_ifnum(ifnum))
+
+    def delete(self, ifnum):
+        url = self.url_prefix + self._ifnum_to_str(ifnum)
+        return self._delete(url)
+
+    def create(self, ifnum, ip_address=None, ip_netmask=None, dhcp=True, enable=True,
```
Comment (Cedev) [inline acos_client/v30/interface.py]: Should the defaults match the box defaults?

File: acos_client/v30/interface.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+               speed="auto"):
+
+        payload = self._build_payload(ifnum=ifnum, ip_address=ip_address, ip_netmask=ip_netmask,
+                                      dhcp=dhcp, enabled=enable, speed=speed)
+        return self._post(self.url_prefix + self._ifnum_to_str(ifnum),
+                          payload)
+
+    def update(self, ifnum, ip_address=None, ip_netmask=None, dhcp=True, enable=True,
```
Comment (Cedev) [inline acos_client/v30/interface.py]: The defaults should be to make no change (None)

**Comment by mdurrant-b3** [comment]
Closing so we can CI as one big bunch.

**Pull request closed without merging**